components/molecule/photoUploader/src/InitialState.js

```diff
@@ -13,6 +13,7 @@ const InitialState = ({
   buttonDesign = BUTTON_DESIGN,
   buttonColor = BUTTON_COLOR,
   buttonText,
+  buttonShape = atomButtonShapes.ROUNDED,
```

Comment:
let the button inherit its default shape

Reply:
done! removed the default value